### PR TITLE
some cleanup

### PR DESCRIPTION
**Pull request by LucaMele**
Fixes # (issue).

Changes proposed in this pull request:

Cleansup the component structure

### CODE REVIEW

File: src/components/a-input/index.js [hunk 1]
```diff
@@ -18,17 +18,17 @@ class AXAFormInput extends BaseComponentGlobal {
...
+      'a-input--inline': inline,
+      'a-input--error': error,
+      'a-input--disabled': disabled,
     });
   }
 }
 
 wcdomready(() => {
-  window.customElements.define('axa-form-input', AXAFormInput);
+  window.customElements.define('axa-input', AXAFormInput);
```
Comment (AndyOGo) [inline src/components/a-input/index.js]: The variable name is still `AXAFormInput` should be just `AXAInput`

File: src/components/a-input/index.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@
 @import "../../styles/control/control-transition";
 @import "./settings/form-input";
```
Comment (AndyOGo) [inline src/components/a-input/index.scss]: this file also has the `form` prefix

File: src/components/o-accordion/index.scss [hunk 3]
```diff
@@ -1,3 +1,3 @@
-.m-accordion {
+.o-accordion {
```
Comment (AndyOGo) [inline src/components/o-accordion/index.scss]: And the file which uses this class?

Comment (LucaMele) [reply src/components/o-accordion/index.scss]: does not exists

Comment (LucaMele) [reply src/components/o-accordion/index.scss]: i remove it

Comment (AndyOGo) [reply src/components/o-accordion/index.scss]: wait, but it should. Thought custom element are by default `inline` and in most cases we want them to be `display: block`.

Comment (LucaMele) [reply src/components/o-accordion/index.scss]: ah yes true sorry i revert :)

File: src/components/a-input/index.js [hunk 4]
```diff
@@ -18,17 +18,17 @@ class AXAFormInput extends BaseComponentGlobal {
...
   }
 }
 
 wcdomready(() => {
-  window.customElements.define('axa-form-input', AXAFormInput);
+  window.customElements.define('axa-input', AXAInput);
 });
 
-export default AXAFormInput;
+export default AXAInput;
```
Comment (AndyOGo) [inline src/components/a-input/index.js]: 👍

File: src/components/o-accordion/index.scss (hunk 3 above)
Comment (LucaMele) [reply src/components/o-accordion/index.scss]: no hold on the aricle underneath has display block as default so is it not enough that?

File: src/components/m-accordion/index.scss [hunk 5]
```diff
@@ -1,3 +0,0 @@
-.m-accordion {
```
Comment (AndyOGo) [inline src/components/m-accordion/index.scss]: please restore, this class is used here:
[`src/components/m-accordion/index.js#L19`](https://github.com/axa-ch/patterns-library/blob/develop/src/components/m-accordion/index.js#L19)

File: src/components/a-input/index.scss [hunk 6]
```diff
@@ -98,7 +98,7 @@
   }
 }
 
-.a-form-input__valid-icon {
+.a-input__valid-icon {
   display: none;
 
   width: $form-input-icon-size;
```
Comment (AndyOGo) [inline src/components/a-input/index.scss]: This variable name should also just start with `$input`

**Pull request merged**